app/main.py

```diff
@@ -1,2 +1,66 @@
+import math
+from typing import Any, Hashable, Final
+
```

Comment:
LOAD_FACTOR = 2/3

Reply:
I mean you can save constant like this and then use it. Read this https://realpython.com/python-constants/